ironfish/src/rpc/routes/wallet/multisig/dkg/round1.test.ts

```diff
@@ -0,0 +1,114 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { multisig } from '@ironfish/rust-nodejs'
+import { createRouteTest } from '../../../../../testUtilities/routeTest'
+
+describe('Route multisig/dkg/round1', () => {
+  const routeTest = createRouteTest()
+
+  it('should create round 1 packages', async () => {
+    const secretName = 'name'
+    await routeTest.client.wallet.multisig.createParticipant({ name: secretName })
+
+    const identity = (await routeTest.client.wallet.multisig.getIdentity({ name: secretName }))
+      .content.identity
+    const otherParticipants = Array.from({ length: 2 }, () => ({
+      identity: multisig.ParticipantSecret.random().toIdentity().serialize().toString('hex'),
+    }))
+    const participants = [{ identity }, ...otherParticipants]
+
+    const request = { secretName, minSigners: 2, participants }
+    const response = await routeTest.client.wallet.multisig.dkg.round1(request)
+
+    expect(response.content).toMatchObject({
+      encryptedSecretPackage: expect.any(String),
```

Comment:
Wonder if it'd be useful to add that this can be decrypted in the test

Reply:
Good idea, will do